mne/epochs.py

```diff
@@ -1556,13 +1556,10 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             If 'truncate', events will be truncated from the end of each event
             list. If 'mintime', timing differences between each event list
             will be minimized.
-        copy : bool
-            If True, a copy of epochs will be returned. Otherwise, the
-            function will operate in-place.
```

Comment:
Do you agree that returning `self` facilitates chaining operations? If so, why do you think it makes no sense to return `self` if `copy=False`?